### DISCUSSION TITLE
Tutorial for Detection Testing?

### DISCUSSION THREAD

**Discussion by BoostN** [blakeblackshear/frigate#4625] [General]
Is it possible to feed a loop of a video into Frigate for testing purposes? I'm trying to get my min_area of a car detection correct and I'm personally tired of driving up and down the driveway. I do have the clip of the video downloaded (From UniFi Protect). Any resources or tutorials?

**Comment by NickM-27** [comment]
https://docs.frigate.video/contributing#2-create-a-local-config-file-for-testing shows how to use a MP4 file as a camera. Be sure to make it's config exactly the same as the original.

**Reply by The-NinthWave** [reply]
> https://docs.frigate.video/contributing#2-create-a-local-config-file-for-testing shows how to use a MP4 file as a camera. Be sure to make it's config exactly the same as the original.

The page is down... If you are a mod, can you put it back, I'd like to do it as well.

Thanks

**Reply by blakeblackshear** [reply]
It's not down. That's just an outdated link. https://docs.frigate.video/development/contributing#2-create-a-local-config-file-for-testing

You could also have used the search built into the documentation site.
